lib/api/controller/base.js

```diff
@@ -632,14 +632,50 @@ class NativeController extends BaseController {
     return null;
   }
 
+  getSearchBody (request) {
+    const searchBodyKeys = [
+      'aggregations',
+      'aggs',
+      'collapse',
+      'explain',
+      'highlight',
+      'query',
+      'search_after',
+      'search_timeout',
+      'sort',
+      '_name',
+      '_source',
+      '_source_excludes',
+      '_source_includes'
+    ];
+    
+    let searchBody = {};
+    if ( request.context.connection.protocol === 'http'
+      && request.input.args.verb
+      && request.input.args.verb === 'GET'
+    ) {
+      searchBodyKeys.forEach(key => {
```

Comment:
Nitpicking: usually we use `for (const ... of ...)` when we are not in a critical code section